fxmanifest.lua

```diff
@@ -1,25 +1,27 @@
 fx_version 'cerulean'
 game 'gta5'
 
-description 'qb-bossmenu'
+description 'qb-management'
 version '2.0.0'
 
+shared_scripts {
+    '@ox_lib/init.lua',
+    '@qb-core/shared/locale.lua',
+	'locales/en.lua',
```

Comment:
```suggestion
	'locales/*.lua',
```

Moving locale support away from just English

Reply:
Readded